akd_core/src/types/node_label/mod.rs

```diff
@@ -79,8 +78,8 @@ impl core::fmt::Display for NodeLabel {
 
 impl NodeLabel {
     /// Hash a [NodeLabel] into a digest, length-prefixing the label's value
-    pub fn hash(&self) -> Digest {
-        crate::hash::hash(&self.to_bytes())
+    pub fn hash(&self) -> Vec<u8> {
```

Comment:
Ah yes, I forgot to add a comment here. The reason why I wanted to change this was because I wantd to eventually get rid of the hashing invocation here as part of #344. That way, we can just return `self.to_bytes()` which is currently a `Vec<u8>`.

I will add the comment.